go/engine/login.go

```diff
@@ -173,7 +174,36 @@ func (e *Login) perUserKeyUpgradeSoft(ctx *Context) error {
 	return err
 }
 
+func (e *Login) checkLoggedInAndNotRevoked(ctx *Context) (bool, error) {
+	loggedInOK, err := e.checkLoggedIn(ctx)
+	if err != nil {
+		return loggedInOK, err
+	}
+	if !loggedInOK {
+		return loggedInOK, nil
+	}
+
+	e.G().Log.Debug("user is logged in, checking if on a revoked device")
+	me, err := libkb.LoadMe(libkb.NewLoadUserForceArg(e.G()))
```

Comment:
can we use the fullselfer here? for people like @malgorithms it's expensive to load me.

Reply:
we should be able to use a `UPAK` for this too, no?

Reply:
It's going to be a bigger change, but ok.

This isn't called often, just on Login.

Reply:
Hmm I don't want to make work.  Will the full-self cacher be OK in this context?

Reply:
What's wrong with FullSelfer?

Reply:
FullSelfer is fine.

Reply:
ok

Reply:
Actually, it fails.  I added a test where the FullSelfer is cached before the reset.  After the reset, the cached user is stale.  

I'm going back to LoadMe.

Reply:
ok, thanks for trying.